api/interservice/notifications/service/events.proto

```diff
@@ -0,0 +1,173 @@
+syntax = "proto3";
+
+package chef.automate.domain.notifications.service;
```

Comment:
FWIW there are actually only three other services that end with the literal `.service` in the package name -- so those are the aberrations rather than the norm.

Reply:
Unfortunately, `chef.automate.domain.notifications` is used for the `api/config` protos for notifications. So this is the most conforming thing I could think of :(